### DISCUSSION TITLE
what are the stars on the stargazer layout for. they are clickable but do nothing.

### DISCUSSION THREAD

**Discussion by retro925** [lightbeam24/CustomTube#92] [Q&A]
![image](https://github.com/lightbeam24/CustomTube/assets/99697031/6875465e-aba5-4a46-8292-42df8fe42f9b)

**Comment by lightbeam24** [comment]
They are not clickable, though I may make them clickable in a future update (no ETA on that). When you hover over the stars, you can see the exact rating:

![Screenshot_2023-10-08_17-00-25](https://github.com/lightbeam24/CustomTube/assets/105595945/48fb89fb-8081-455e-af2f-75cafdbaf911)

Thinking of adding like and dislike counts to this tooltip. Not sure why I used the word "upvoted" though, will probably change that.

Similarly, this also happens when you hover over the like to dislike bar (though 2015-2017 has some layering issues so it often doesn't work):

![Screenshot_2023-10-08_17-03-29](https://github.com/lightbeam24/CustomTube/assets/105595945/a414b97c-77e5-4441-a668-3e170d42a542)

**Comment by bangbangyouall** [comment]
i thought that was a pun
